contest/remote/kunit.py

```diff
@@ -94,16 +94,17 @@ def summary_flat(expected, got, sub_path=""):
     bad_tests = []
     for case in got["test_cases"]:
         code = str_to_code[case["status"]]
+        name = namify(case["name"])
 
-        exp = expected.get(got["name"], {}).get(case["name"])
+        exp = expected.get(got["name"], {}).get(name)
```

Comment:
mmh, maybe I should not do the conversion here. (I'm not sure what's in `kunit-expected` file)